Gemfile

```diff
@@ -21,4 +21,5 @@ group :development do
   gem "guard-rake"
   gem "pry"
   gem "yard"
+  gem "vagrant-wrapper"
```

Comment:
I've had problems previously with vagrant-wrapper, and had support issues with other modules. This shouldn't be required.